config_src/drivers/timing_tests/time_MOM_remapping.F90

```diff
@@ -0,0 +1,93 @@
+program time_MOM_remapping
+
+! This file is part of MOM6. See LICENSE.md for the license.
+
+use MOM_remapping, only : remapping_CS
+use MOM_remapping, only : initialize_remapping
+use MOM_remapping, only : remapping_core_h
+
+implicit none
+
+type(remapping_CS) :: CS
+integer, parameter :: nk=75, nij=20*20, nits=10, nsamp=100, nschemes = 2
+character(len=10) :: scheme_labels(nschemes)
+real, dimension(nschemes) :: timings, tmean, tstd, tmin, tmax
```

Comment:
Please add comments describing the 13 real variables on the next 3 lines, along with their units.